### PR TITLE
fix: [PIE-1808]: Fix Failure color, and graph alignment

### PR DESCRIPTION
**Pull request by ishantmehtaharness**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by ishantmehtaharness** [comment]
trigger cypress

File: src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx [hunk 1]
```diff
@@ -238,6 +238,11 @@ const getFormattedNumber = (givenNumber?: number | string): string => {
 }
 
 const summaryCardRenderer = (cardData: SummaryCardData, groupByValue: string): JSX.Element => {
+  let color = cardData.trend.includes('-') ? Color.RED_500 : Color.GREEN_500
+  // Failure should be in Red
+  if (cardData.title === 'Failure Rate') {
```
Comment (chiragp1212) [inline src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx:243]: why are we depending on "title" and hyphens to set color? Shouldn't this be based on some status?

File: src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx [hunk 2]
```diff
@@ -585,7 +586,7 @@ function LandingDashboardDeploymentsWidget(): React.ReactElement {
           </Layout.Horizontal>
           <div className={css.mostActiveServicesChartContainer}>
             <StackedSummaryTable
-              barLength={185}
+              // barLength={185}
```
Comment (chiragp1212) [inline src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx:589]: unwanted comment

File: src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx [hunk 3]
```diff
@@ -238,6 +238,11 @@ const getFormattedNumber = (givenNumber?: number | string): string => {
 }
 
 const summaryCardRenderer = (cardData: SummaryCardData, groupByValue: string): JSX.Element => {
+  let color = cardData.trend.includes('-') ? Color.RED_500 : Color.GREEN_500
```
Comment (chiragp1212) [inline src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx:241]: we can memoise this color variable

File: src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx (hunk 1 above)
Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx:243]: such handlings are done using `cardData.title` in this area. Plan is to add enums later

File: src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx (hunk 2 above)
Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx:589]: We might need it back. so commented

File: src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx (hunk 3 above)
Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx:241]: This is just a ternary operator. do we optimise such things?

**Review by chiragp1212 — APPROVED** [review]

File: src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx (hunk 3 above)
Comment (chiragp1212) [reply src/modules/70-pipeline/components/LandingDashboardDeploymentsWidget/LandingDashboardDeploymentsWidget.tsx:241]: any optimisation is welcome, but yes it doesn't make too much of a difference here

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**